classes/controller/AdminController.php

```diff
@@ -3720,28 +3732,6 @@ public function validateRules($class_name = false)
 
         /* Overload this method for custom checking */
         $this->_childValidation();
-
-        /* Checking for multilingual fields validity */
-        if (isset($rules['validateLang']) && is_array($rules['validateLang'])) {
```

Comment:
This code is dead since 1.5.0.1
The ObjectModel class should be cleaned here:
https://github.com/PrestaShop/PrestaShop/blob/b96a10089eb989efa83bcdca274d9f68c4ae5223/classes/ObjectModel.php#L127